From be1d59e6212837514e8cbcde7fecdda1cc7494e3 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 25 Oct 2017 04:31:42 +0300
Subject: [PATCH 2057/2241] crypto: inside-secure: use min_t instead of min

better to use min_t for same type variables

Change-Id: I251faeda3df6c6ec199556936b318c403cb8e3b4
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/45572
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/crypto/inside-secure/safexcel.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index 533474a..08892ca 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -740,7 +740,7 @@ finalize:
 		priv->ring[ring].busy = 1;
 
 		val = EIP197_HIA_RDR_THRESH_PKT_MODE |
-			EIP197_HIA_RDR_THRESH_PROC_PKT(min(priv->ring[ring].egress_cnt,
+			EIP197_HIA_RDR_THRESH_PROC_PKT(min_t(int, priv->ring[ring].egress_cnt,
 							   EIP197_MAX_BATCH_SZ));
 		writel(val, EIP197_HIA_AIC_xDR(priv) + EIP197_HIA_RDR(ring) + EIP197_HIA_xDR_THRESH);
 	}
@@ -946,7 +946,7 @@ more_results:
 	spin_lock_bh(&priv->ring[ring].egress_lock);
 
 	/* get the pending request count */
-	egress_cnt = min(priv->ring[ring].egress_cnt, EIP197_MAX_BATCH_SZ);
+	egress_cnt = min_t(int, priv->ring[ring].egress_cnt, EIP197_MAX_BATCH_SZ);
 
 	if (!egress_cnt) {
 		/* no more request in ring */
-- 
2.7.4

